L/lrslib/build_tarballs.jl

```diff
@@ -25,10 +25,11 @@ if [[ $target == i686* ]] || [[ $target == arm* ]]; then
 fi
 
 if [[ $target == *apple* ]]; then
+  export CC=gcc
   sed -i -e 's#-Wl,-soname=#-install_name #' makefile
   extraargs="SONAME=liblrs.0.dylib SHLINK=liblrs.dylib SHLIB=liblrs.0.0.0.dylib"
 elif [[ $target == *freebsd* ]]; then
-  export CC="$CC $LDFLAGS"
+  export CC="gcc $LDFLAGS"
```

Comment:
Now that I see this, `LDFLAGS` isn't set anywhere
```suggestion
  export CC="gcc"
```

Reply:
Does [this](https://docs.binarybuilder.org/dev/build_tips/#Automatic-environment-variables) doc not mean BinaryBuilder sets them?

Reply:
> These are other environment variables that you may occasionally need to set during a build

The `*FLAGS` variables aren't set.  They're mentioned because most build system are influenced by them and sometimes you do need to set them.  But here `LDFLAGS` isn't set anywhere.